nextcloud/Containerfile

```diff
@@ -0,0 +1,53 @@
+FROM dir://base-buster
+ARG VERSION=16
+
+RUN apt-get -y --no-install-recommends install \
+	    wget \
+	    php7.3-fpm \
+	    php7.3 \
+	    php7.3-gd \
+	    php7.3-intl \
+	    php7.3-pgsql \
+	    php7.3-apcu \
+	    php7.3-curl \
+	    php7.3-memcache \
+	    php7.3-redis \
+	    php7.3-ldap \
+	    php7.3-xml \
+	    php7.3-zip \
+	    php7.3-json \
+	    php7.3-mbstring \
+	    strace \
+	    bzip2 \
+	    git \
+	    php-imagick; \
+    mkdir /var/www ; \
+    wget -qO- https://download.nextcloud.com/server/releases/latest-${VERSION}.tar.bz2 | \
+    tar -C /var/www -xjv --no-same-owner; \
+    chown www-data /var/www/nextcloud/apps; \
+    chown www-data /var/www/nextcloud/config
+
+RUN DIR="/var/www/nextcloud/apps/richdocuments"; \
+    git clone https://github.com/nextcloud/richdocuments.git $DIR ; \
+    git --git-dir=$DIR/.git --work-tree=$DIR checkout $(git --git-dir=$DIR/.git --work-tree=$DIR tag | tail -n1)
```

Comment:
clever, but I am not sure the ordering is correct (current versions have a "v" prefix in the version number some older releases not)

Reply:
Now it should in principle work